server/methods/customerService.js

```diff
@@ -0,0 +1,60 @@
+Meteor.methods({
+  'advancedFulfillment/cancelOrder': function (orderId, userId) {
+    check(orderId, String);
+    check(userId, String);
+    let history = {
+      event: 'orderCanceled',
+      userId: userId,
+      updatedAt: new Date()
+    };
+    ReactionCore.Collections.Orders.update({
+      _id: orderId
+    }, {
+      $addToSet: {
+        history: history
+      },
+      $set: {
+        'advancedFulfillment.workflow.status': 'orderCanceled',
+        'advancedFulfillment.impossibleShipDate': false
+      }
+    });
+  },
+  'advancedFulfillment/bundleColorConfirmation': function (orderId, userId) {
+    check(orderId, String);
+    check(userId, String);
+    let history = {
+      event: 'bundleColorConfirmed',
+      userId: userId,
+      updatedAt: new Date()
+    };
+    ReactionCore.Collections.Orders.update({
+      _id: orderId
+    }, {
+      $addToSet: {
+        history: history
+      },
+      $set: {
+        bundleMissingColor: false
+      }
+    });
+  },
+  'advancedFulfillment/updateSkiPackageWithCustomerInfo': function (orderId, userId, skiId, age, shoeSize, level) {
```

Comment:
We do need to start making sure we are checking for proper roles in these methods - especially these methods where we are taking params and putting them directly into the database.